src/modules/item/reducer.ts

```diff
@@ -383,6 +373,19 @@ export function itemReducer(state: ItemState = INITIAL_STATE, action: ItemReduce
             }
           }
         }
+        case PUBLISH_THIRD_PARTY_ITEMS_SUCCESS: {
+          const items: Item[] = transaction.payload.items
+          return {
+            ...state,
+            data: {
+              ...state.data,
+              ...items.reduce((accum, item) => {
+                accum[item.id] = item
+                return accum
+              }, {} as ItemState['data'])
+            }
+          }
+        }
```

Comment:
Yeah, but since we're not using the `isPublished` field for the TP items, I thought it might not be necessary to explicitly update the `isPublished` field. The `PUBLISH_THIRD_PARTY_ITEMS_SUCCESS` action will update the item curations so the logic to know if it's public will be there.
That said, you're right, could re-use the `PUBLISH_COLLECTION_SUCCESS` and it should be the same

Reply:
If `isPublished` is kind of irrelevant, I would move it up to `PUBLISH_COLLECTION_SUCCESS` as it will avoid you from creating a test for this new behavior in the reducer and it will also make the code shorter.

Reply:
Agreed, moved!